crates/rust-analyzer/src/to_proto.rs

```diff
@@ -862,6 +865,141 @@ pub(crate) fn runnable(
     })
 }
 
+pub(crate) fn annotation(
+    snap: &GlobalStateSnapshot,
+    annotation: Annotation,
+) -> Result<lsp_types::CodeLens> {
```

Comment:
```suggestion
pub(crate) fn code_lens(
    snap: &GlobalStateSnapshot,
    annotation: Annotation,
) -> Result<lsp_types::CodeLens> {
```

the functions are named after the return type.